lib/px/utils/pxcommonutils.lua

```diff
@@ -0,0 +1,21 @@
+--
```

Comment:
@yaronpx pxtimer uses to measure time outside the application scope, adding more logic to this file was not a good practice to implement it there

once it was crashing I didnt want to start and modify this file, I decided to move it to a util class so later we can add more utils there